graphql/schemabuilder/pagination.go

```diff
@@ -436,6 +447,51 @@ func (c *connectionContext) applyTextFilter(nodes []interface{}, args Pagination
 	return filteredNodes, nil
 }
 
+func (c *connectionContext) applySort(nodes []interface{}, args PaginationArgs) ([]interface{}, error) {
+	if args.SortBy == nil {
+		return nodes, nil
+	}
+
+	// Default to ascending sort.
+	sortOrder := SortOrder_Ascending
+	if args.SortOrder != nil {
+		sortOrder = *args.SortOrder
+	}
+
+	sortField, ok := c.SortFields[*args.SortBy]
+	// If the field wasn't registered, it's an unknown sort field.
+	if !ok {
+		return nil, fmt.Errorf("Unknown sort field %s", *args.SortBy)
+	}
+
+	// sortValues is the slice we'll be sorting (with the sorted values) in order to figure out
+	// node order.
+	sortValues := make([]sortReference, len(nodes))
+	for i, node := range nodes {
+		// Resolve the graphql.Field made for sorting.
+		sortValue, err := sortField.Resolve(context.TODO(), node, nil, nil)
```

Comment:
same note here: let's figure out how to share a context for the computation cache

Reply:
This is done in another commit